test/Microsoft.Azure.WebJobs.Host.UnitTests/PublicSurfaceTests.cs

```diff
@@ -61,13 +62,44 @@ public void LoggingPublicSurface_LimitedToSpecificTypes()
         }
 
         [Fact]
-        public void WebJobsHostPublicSurface_LimitedToSpecificTypes()
+        public void WebJobs_VerifyPublicSurfaceArea()
+        {
+            // The DLL containing the binding attributes should be truly minimal and have no extra dependencies. 
+            var assembly = typeof(AutoResolveAttribute).Assembly;
+
+            var expected = new[]
+            {
+                "AppSettingAttribute",
```

Comment:
During the recent refactorings, it appears that we lost the tests for the WebJobs core assembly